spec/Filter/Filter.BinaryComparison.Implementations.js

```diff
@@ -0,0 +1,49 @@
+describe('Filter.BinaryComparison', function() {
```

Comment:
More complex tests with defined operands are really needed, you just separately test that filters returns expected wrappers, but you don't test full filter use cases.